builder/bundles.go

```diff
@@ -638,7 +638,7 @@ func (b *Builder) buildBundles(set bundleSet, downloadRetries int) error {
 	}
 
 	if _, ok := set[b.Config.Swupd.Bundle]; !ok {
-		return fmt.Errorf("couldn't find bundle %q specified in configuration as the update bundle",
+		fmt.Printf("Warning: couldn't find bundle %q specified in configuration as the update bundle. Bundle won't be configured.\n",
```

Comment:
Rather than having this warning, we still check if `b.Config.Swupd.Bundle` is `""`. This would indicate that there should be no update bundle. Then, whenever we use mixer for 3rd-party, we change config so that Swupd.BUNDLE is empty.

Reply:
That was my first implementation, but I thought that this would be more strait forward. If you remove the os-core-update from the mix, we don't update that. If you add it, we update.
So it's about the bundles that are selected from the user, not the name of the bundle that should include swupd.
But I'm ok to change that too, if you think it's going to be less error prone